client/src/Tools/FetchData/fetchContestsfunctions.js

```diff
@@ -0,0 +1,16 @@
+import { requestOptionsGET } from './requestOptions';
+
+export async function getContestsCards(taker) {
+  let url = '/api/contests/card/data';
+  if (taker) {
+    url = `/api/contests/card/data?taker=${taker}`;
+  }
```

Comment:
you could also concat the taker part or use a ternary expression, just an idea

Reply:
in that case I feel safety with if :) with ternary looks here a little bit out of mine control. Just a feeling

Reply:
:) Well, ifs are fine, as long as there arent too many